app/collections/show.cjsx

```diff
@@ -41,23 +41,23 @@ CollectionPage = React.createClass
     <PromiseRenderer promise={@props.collection.get('owner')}>{(owner) =>
       [ownerName, name] = @props.collection.slug.split('/')
       params = {owner: ownerName, name: name}
-
+      baseLink = "/collections/#{ownerName}/#{name}"
```

Comment:
`params` above is unused, which means `ownerName` and `name` are unused except for here, where it'd be more appropriate to use `@props.collection.slug` anyway.